ui/src/views/tools/SelectVmwareVcenter.vue

```diff
@@ -281,52 +256,17 @@ export default {
     listZoneVmwareDcs () {
       this.loading = true
       api('listVmwareDcs', { zoneid: this.sourcezoneid }).then(response => {
-        if (response.listvmwaredcsresponse.vmwaredc && response.listvmwaredcsresponse.vmwaredc.length > 0) {
-          this.existingvcenter = response.listvmwaredcsresponse.vmwaredc
+        if (response.listvmwaredcsresponse.VMwareDC && response.listvmwaredcsresponse.VMwareDC.length > 0) {
+          this.existingvcenter = response.listvmwaredcsresponse.VMwareDC
```

Comment:
@DaanHoogland seems this is the issue for vmware dc listing in zone

Reply:
That is in VmwareData.vue @sureshanaparti but this change is also needed import VMware feature.

Reply:
correct, @DaanHoogland check if the object name change is requried or not.